src/main/scripts/startMZmine_Windows.bat

```diff
@@ -52,16 +52,6 @@ mkdir %TMP_FILE_DIRECTORY%
 
 rem Set R environment variables.
 set R_HOME=C:\Program Files\R\R-2.12.0

```

Comment:
Is the R_HOME variable still required? 
I thought you had implemented some auto-detection of R location, haven't you?